src/quo/components/buttons/slide_button/view.cljs

```diff
@@ -26,8 +26,9 @@
                              (reanimated/set-shared-value x-pos clamped-x)
                              (when (and reached-end? (not sliding-complete?))
                                (set-sliding-complete true)
+                               (js/setTimeout reset-fn 500)
```

Comment:
could you please elaborate

Reply:
we used to do this timed out reset with the on-complete prop inside `standard-auth/slide-button`, but most usages of slide button were through standard-auth anyways and it tied us to using the standard-auth version of slide button. Now though we have places where authorization shouldn't happen when sliding is complete, but as a result of e.g. a signal triggered by sliding. This makes the standard-auth slide button a bit useless, so this move was to make it easier to use the quo slide button without losing the reset functionality.